package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "garmin-node-api",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Module for logging in to Garmin Connect and retrieving basic data",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "Jake Partusch",
+  "author": "Jake Partusch, Michael Svanström",
```

Comment:
I'd be happy to add you under "contributors" :)

https://docs.npmjs.com/files/package.json#people-fields-author-contributors